docusaurus/docs/license.md

```diff
@@ -6,6 +6,7 @@ sidebar_position: 8
 
 Copyright (C) 2017-2023 [The ORT Project Authors](https://github.com/oss-review-toolkit/ort/blob/main/NOTICE).
 
+OSS Review Toolkit (ORT) is licensed under the Apache License, Version 2.0.
```

Comment:
Why isn't the link directly below this line enough, as the docs are also hosted in the repo the link points to?

Reply:
Unfortunately no, as the moment that i show to some persons, they asked why the license is not mention there.
And is really clear for persons to need to do a one extra jump to a not user friendly page of github as non-developers.

Reply:
I agree that the website should mention the license directly without requiring to click on a link.

Reply:
Done

Reply:
Ok, still it feels like duplication what we have here now. How about doing

    OSS Review Toolkit (ORT) is licensed under the [Apache License, Version 2.0](https://github.com/oss-review-toolkit/ort/blob/main/LICENSE).

and omitting the "See the LICENSE" sentence from below then?

Reply:
Nope, because you need something indicating for the full license text.
This is a normal practice everywhere. A good example would be the projects page in github, where the main license ( or conjunction ) is displayed on the right, with a deep link for the full file.

Reply:
I'm not following you. The full license text is still linked. It's just that the link is now called "Apache License, Version 2.0" instead of "LICENSE".